rosdep/base.yaml

```diff
@@ -6202,6 +6202,20 @@ libvtk-qt:
     xenial: [libvtk6-qt-dev]
     yakkety: [libvtk6-qt-dev]
     zesty: [libvtk6-qt-dev]
+libvtk9-dev:
+  debian: [libvtk9-dev]
+  ubuntu:
+    '*': [libvtk9-dev]
+    bionic: null
+    focal: null
+libvtk9.1:
```

Comment:
Hm.  I know that this is the name of the Debian package, but this seems like far too specific a name.  That is, it is unclear to me that we would ever be able to fill in other entries for this, since a newer Debian/Ubuntu version might add a new version, and Fedora (for instance) might also have a different version.

Could we maybe instead add this to the `libvtk` key above?  That one already has different versions in it.

Reply:
Yeah, that seems like a good compromise here.  I'd be OK with that change.

Reply:
removed the libvtk9.1 rule in 80c2049